src/unstract/sdk/adapters/vectordb/no_op/src/no_op_vectordb_adapter.py

```diff
@@ -0,0 +1,78 @@
+import os
+import time
+from typing import Any
+
+from llama_index.core.schema import BaseNode
+from llama_index.core.vector_stores.types import VectorStore
+
+from unstract.sdk.adapters.exceptions import AdapterError
+from unstract.sdk.adapters.vectordb.constants import VectorDbConstants
+from unstract.sdk.adapters.vectordb.helper import VectorDBHelper
+from unstract.sdk.adapters.vectordb.no_op.src.no_op_vectordb import NoOpVectorDB
+from unstract.sdk.adapters.vectordb.vectordb_adapter import VectorDBAdapter
+
+
+class NoOpVectorDBAdapter(VectorDBAdapter):
```

Comment:
@harini-venkataraman Better to rename this to `NoOpVectorDB`.

Similarly we can do so in other places.